javaslang/src-gen/main/java/javaslang/CheckedFunction0.java

```diff
@@ -93,6 +93,18 @@
     }
 
     /**
+     * Narrows the given {@code CheckedFunction0<? extends R>} to {@code CheckedFunction0<R>}
+     *
+     * @param wideFunction A {@code CheckedFunction0}
+     * @param <R> return type
+     * @return the given {@code wideFunction} instance as narrowed type {@code CheckedFunction0<R>}
+     */
+    @SuppressWarnings("unchecked")
+    static <R> CheckedFunction0<R> narrow(CheckedFunction0<? extends R> wideFunction) {
```

Comment:
I've changed parameter name to `f`, should I change parameter name for `lift` also?